crates/primitives/src/user_operation/mod.rs

```diff
@@ -440,11 +470,71 @@ pub struct UserOperationReceipt {
     pub tx_receipt: TransactionReceipt,
 }
 
+#[derive(Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RpcUserOperation {
```

Comment:
Rename to `UserOperationRpc` ? I don't feel strongly about that, but other user operation names have suffix